packages/circularprogress/package.json

```diff
@@ -9,20 +9,20 @@
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
   "scripts": {
-    "build": "run-s build:js build:cjs build:css",
-    "build:cjs": "babel --root-mode upward src --out-dir dist/cjs && cp src/index.d.ts dist/cjs/index.d.ts",
+    "build": "run-s build:cjs build:esm build:css",
+    "build:cjs": "tsc --project tsconfig.build.json --module commonjs --target es5 --outDir dist/cjs",
     "build:css": "build-css --useGlamor -i dist/cjs/css/index.js",
-    "build:js": "cross-env ESM=true babel --root-mode upward src --out-dir dist/esm && cp src/index.d.ts dist/esm/index.d.ts",
+    "build:esm": "tsc --project tsconfig.build.json --module es2015 --target es5 --outDir dist/esm",
     "build:watch": "yarn build:js -- --watch",
     "storybook": "start-storybook -p 6006",
     "test": "tsc --noEmit --project ./tsconfig.json && jest",
-    "test:watch": "yarn test -- --watchAll"
+    "test:watch": "yarn test --watchAll"
   },
```

Comment:
please add the reference for types

Reply:
^ I don't know what you mean.

> please add the reference for types

Please elaborate.  What reference? What types? I'm not intuiting that from the line reference here.

Reply:
we've been adding these two lines to each package. one for types and one for styles just because

```
  "style": "dist/index.css",
  "types": "dist/esm/index.d.ts",
```